podcasts/SwiftUI/ContentSizeGeometryReader.swift

```diff
@@ -0,0 +1,64 @@
+import SwiftUI
+
+/// A GeometryReader wrapper view that calculates the size of the content and adjusts the
+///
+///
+/// The contentSizeUpdated to be informed of the new content size outside of SwiftUI
```

Comment:
Nipticky: extra `The` and newline perhaps?

Reply:
Oops! Fixed here: 3f1e4857e926a6e763aefd1e5166a3265ecc4883